src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -39,7 +39,19 @@ <h5 class="col-md-6 col-sm-6 col-xs-8 no-padding truncate">
                                       {{comment.relationalData?.creator?.attributes?.fullName}}
                                   </strong>
                               </h5>
-                               <p class="col-md-12 col-xs-12 col-sm-12 no-padding" id="{{'comment_body_' + counter}}">{{comment.attributes.body}}</p>
+                              <p *ngIf="!loggedIn || (currentUser.id !== comment.relationalData.creator?.id)"
+                                  id="{{'comment_body_' + counter}}"
+                                  class="col-md-12 col-xs-12 col-sm-12 no-padding">
+                                {{ comment.attributes.body }}
+                              </p>
+                              <p  *ngIf="!!loggedIn && (currentUser.id === comment.relationalData.creator?.id)"
```

Comment:
UXD specified editable interface has many inconsistencies, so we made a functional implementation like this. For now, this should suffice for now.

@SMahil is working on some changes on comments UI and if we need more then we can have them implemented later when we have better clarity.